cmd/cmount/mount.go

```diff
@@ -135,6 +141,22 @@ func waitFor(fn func() bool) (ok bool) {
 	return false
 }
 
+// findUnusedDriveLetter searches mounted drive list on the system
```

Comment:
We have some code which does this already here:

https://github.com/rclone/rclone/blob/30c8b1b84f548f08740148c6e9a086146af1f60d/vfs/vfstest/fs.go#L157-L165

Maybe we should factor your code out into `lib/file` (then we can have a non-windows version which always returns "") and call it from both places?